README.md

```diff
@@ -71,54 +126,39 @@ inet:port_number()}`, are used to discover the rest of the cluster.
 
 For options, see [Options](#options) below.
 
-### `connect/3`
-
-```Erlang
-connect(addr(), port(), [client_opt()]) -> {ok, client_ref()} | {error, term()}.
-```
-
-Connects to a single node. The process is supervised by the `ered`
-application, which needs to be started in advance.
-
-For options, see [Client options](#client-options) below.
-
 ### `close/1`
 
 ```Erlang
-close(ered_ref()) -> ok.
+ered_cluster:close(cluster_ref()) -> ok.
```

Comment:
This one example looked good.
I think we can add the module to each API description since the `ered` and `ered_cluster` apis looks similar.
It's a bit mixed right now.